Frontend/ui-customer/src/Screens/RideBookingFlow/RiderRideCompletedCard/ComponentConfig.purs

```diff
@@ -28,5 +33,64 @@ primaryButtonConfig state = PrimaryButton.config
     , isClickable = not state.ratingCard.recordAudioState.isRecording
     }
 
+primaryButtonConfigForCarousel :: PrimaryButton.Config
+primaryButtonConfigForCarousel = PrimaryButton.config
+    { textConfig
+        { text = getString LT.DONE
+        , color = Color.yellow900
+        , accessibilityHint = "Done with Carousel" 
+        }
+    , alpha = 1.0
+    , background = Color.black900
+    , margin = (Margin 0 0 0 0)
+    , id = "DoneWithCarousel"
+    , enableRipple = true
+    , rippleColor = Color.rippleShade
+    , isClickable = true
+    }
+
 driverInfoCardConfig :: RiderRideCompletedScreenState -> FavouriteDriverInfoCard.FavouriteDriverInfoCardState
-driverInfoCardConfig state = FavouriteDriverInfoCard.config 
\ No newline at end of file
+driverInfoCardConfig state = FavouriteDriverInfoCard.config 
+
+issueReportBannersTransformer  :: Array CustomerIssueCard -> Array (Record IssueReportBannerProps)
```

Comment:
Can you confirm if these functions are present at two places? If so We can remove from other or use that only